src/main/kotlin/com/team4099/robot2022/subsystems/Drivetrain.kt

```diff
@@ -35,42 +34,26 @@ object Drivetrain : SubsystemBase() {
   val wheels =
       listOf(
           Wheel(
-              CANSparkMax(
-                  Constants.Drivetrain.FRONT_LEFT_DIRECTION_ID,
-                  CANSparkMaxLowLevel.MotorType.kBrushless),
-              CANSparkMax(
-                  Constants.Drivetrain.FRONT_LEFT_SPEED_ID,
-                  CANSparkMaxLowLevel.MotorType.kBrushless),
+              TalonFX(Constants.Drivetrain.FRONT_LEFT_DIRECTION_ID),
+              TalonFX(Constants.Drivetrain.FRONT_LEFT_SPEED_ID),
               CANCoder(Constants.Drivetrain.FRONT_LEFT_CANCODER_ID),
```

Comment:
not using CANcoders this year, switch to `AnalogPotentiometer` with correct parameters

```
AnalogPotentiometer​(AnalogInput input,
double fullRange,
double offset)
``` 
is the constructor definition, where `fullRange` for us is the full rotation (so `2*PI`) and we don't care about offset here (so it can just be 0), we can just use the offset variable we already have defined one line below this